ui/src/components/TestCases/SepGroupTracker.tsx

```diff
@@ -0,0 +1,10 @@
+export const SepGroupTracker: React.FC<{
```

Comment:
this is just a placeholder component to show the progress UI. This is will be styled in a subsequent PR